app/public/stylesheets/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+ font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+ background-color: #40E0D0;
+ overflow-x: hidden;
+}
+
+h1 {
+  text-align: center;
+  width: 100%;
+  color: white;
+  font-weight: bold;
+}
+
+#formBox {
+  min-height: 500px;
+  width: 100%;
+  max-width: 750px;
+  padding: 25px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#form {
+  width: 95%;
+  padding: 15px;
+  max-width: 500px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#form:after{
+  content: ".";
+  visibility: hidden;
+  display: block;
+  height: 0;
+  clear: both;
+}
+
+#form > input {
```

Comment:
I would recommend combining the CSS for all of the direct descendants of the #form element.

Something like this would make your CSS file a bit smaller:

`#form > select, #form > button, #form > input {
    float: left;
    clear: both;
    height: 35px;
    width: 100%;
    border-radius: 20px;
    margin-bottom: 5px;
    padding-left: 10px;
}`

This would allow you to combine many of the css properties that are the same between multiple css selectors.